processor/src/main/java/org/mapstruct/ap/internal/conversion/Conversions.java

```diff
@@ -233,12 +233,15 @@ private void registerJava8TimeConversions() {
         register( Period.class, String.class, new StaticParseToStringConversion() );
         register( Duration.class, String.class, new StaticParseToStringConversion() );
 
-        // Java 8 to Date
+        // Java 8 time to Date
         register( ZonedDateTime.class, Date.class, new JavaZonedDateTimeToDateConversion() );
         register( LocalDateTime.class, Date.class, new JavaLocalDateTimeToDateConversion() );
         register( LocalDate.class, Date.class, new JavaLocalDateToDateConversion() );
         register( Instant.class, Date.class, new JavaInstantToDateConversion() );
 
+        // Java 8 time (internal)
```

Comment:
What do you mean with "(internal)"?

Reply:
Oh, what I meant was we are converting from a Java 8 type to another Java 8 type, i.e. "internally" within the same package. I've cleaned that up now :)